pipeline/scripts/update-latest-yml.js

```diff
@@ -6,6 +6,7 @@ const YAML = require('js-yaml');
 const path = require('path');
 
 const parentDir = process.argv[2];
+const platform = process.argv.length > 3 ? process.argv[3] : process.platform;
 
 const getLatestYAMLPath = parentDir => {
     const platformModifier = process.platform == 'darwin' ? '-mac' : process.platform == 'linux' ? '-linux' : '';
```

Comment:
this could also change to stop relying on process.platform (to enable running this agent on any build machine, for instance)  but I've left it out for now